vnext/local-cli/runWindows/utils/commandWithProgress.js

```diff
@@ -50,18 +71,23 @@ function commandWithProgress(spinner, taskDoingName, command, args, verbose) {
     }
 
     const cp = child_process.spawn(command, args, spawnOptions);
-
+    var error = null;
     if (!verbose) {
       cp.stdout.on('data', chunk => {
         const text = chunk.toString();
         setSpinnerText(spinner, taskDoingName + ': ', text);
       });
       cp.stderr.on('data', chunk => {
         const text = chunk.toString();
+        if (!error) {
```

Comment:
This logic is a bit confusing to me.

`(!error)` should always be true bc we init it to null and don't touch it before this line. Why don't we just ouput text and not introduce an extra variable?

Reply:
!null == true

---
In reply to: [405940047](https://github.com/microsoft/react-native-windows/pull/4536#discussion_r405940047) [](ancestors = 405940047)

Reply:
I mean this condition will never be false.

Reply:
Oh, we mutate error on subsequent callbacks.

Reply:
Could we name this variable something to make this a bit clearer? Like `firstErrorLine`?

Reply:
sure thing

---
In reply to: [405943833](https://github.com/microsoft/react-native-windows/pull/4536#discussion_r405943833) [](ancestors = 405943833)

Reply:
yep

---
In reply to: [405943004](https://github.com/microsoft/react-native-windows/pull/4536#discussion_r405943004) [](ancestors = 405943004)